images/linux/scripts/tests/Browsers.Tests.ps1

```diff
@@ -16,4 +16,10 @@ Describe "Chrome" {
     It "Chrome Driver" {
         "chromedriver --version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "Chromium" -Skip:(-not (Test-IsUbuntu20)) {
```

Comment:
We don't need this test after moving the installation to apt, but, please pay attention that if the executable name differs from the package name then you need to add a condition like this one
https://github.com/actions/virtual-environments/blob/a0656ad9b456750dfb92253c17669b6d4d882b3f/images/linux/scripts/tests/Apt.Tests.ps1#L12-L15

Reply:
We've decided to leave the separate test for chromium browser.